### DISCUSSION TITLE
Installing pre/post processing tools

### DISCUSSION THREAD

**Discussion by markyoder** [SeisSol/SeisSol#724] [General]
Ok never mind -- but it's probably not a bad idea to leave a post here as a reference to others. Basically, I was banging my head against compiling, `preprocessing/meshing/gmsh2gambit`.

The short version is that the `utils` link/subdirectory is missing from that source repo; there is a link back to it in `preprocessing/meshing/gambit2netcdf/` .

So if you get a, " can't find the `utils/args.h` header file. " when you are compiling this, that header is part of this:

[https://github.com/TUM-I5/utils](https://github.com/TUM-I5/utils)

Which you should find in `SeisSol/submodules/utils`, or you can clone/copy it to your source directory.

Own question(s) answered...

**Comment by NicoSchlw** [comment]
Hi, I think these tools are not really up-to-date anymore but are now included in PUMGen:
https://seissol.readthedocs.io/en/latest/meshing-with-pumgen.html
